core/templates/pages/admin-page/misc-tab/admin-misc-tab.directive.ts

```diff
@@ -130,71 +121,71 @@ angular.module('oppia').directive('adminMiscTab', [
         };
 
         ctrl.sendDummyMailToAdmin = function() {
-          $http.post(SEND_DUMMY_MAIL_HANDLER_URL)
+          AdminMiscTabBackendApiService.sendDummyMail()
             .then(function(response) {
               ctrl.setStatusMessage('Success! Mail sent to admin.');
             }, function(errorResponse) {
+              console.log(errorResponse)
```

Comment:
I was expecting pre-push lint checks to catch this, can you confirm that the pre-push hooks are working for you? (i.e, when you run `git push ...` command the linter and frontend tests runs before pushing the changes.)

Reply:
@DubeySandeep, I think they have not triggered automatically. Any suggestions on how to resolve this?

Reply:
Its working fine now